CMakeLists.txt

```diff
@@ -19,7 +19,6 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 add_cxx_flag("-Wall")
-add_cxx_flag("-Wno-deprecated")
```

Comment:
I'd rather keep `-Wno-deprecated` to prevent people from re-introducing deprecated functions/classes.